Source/cmake/OptionsGTK.cmake

```diff
@@ -8,7 +8,7 @@ SET_PROJECT_VERSION(2 51 0)
 set(USER_AGENT_BRANDING "" CACHE STRING "Branding to add to user agent string")
 
 # Update Source/WTF/wtf/Platform.h to match required GLib versions.
-find_package(GLIB 2.70.0 REQUIRED COMPONENTS gio gio-unix gobject gthread gmodule)
+find_package(GLib 2.70.0 REQUIRED COMPONENTS Gio GioUnix Object Thread Module)
```

Comment:
Shouldn't `GioUnix` imply `Object` and `Gio`. 

Same elsewhere.

Reply:
Good point! I changed this before having the idea of adding the support to declare the dependencies among modules. We should be able to remove `Object` and `Gio` from the list here, I'll try.